src/AWS.Deploy.CLI/Commands/GenerateDeploymentProjectCommand.cs

```diff
@@ -0,0 +1,185 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.IO;
+using System.Threading.Tasks;
+using AWS.Deploy.Common;
+using AWS.Deploy.Orchestration;
+using AWS.Deploy.Orchestration.CDK;
+using AWS.Deploy.Orchestration.Data;
+using AWS.Deploy.Orchestration.Utilities;
+using AWS.Deploy.Recipes;
+
+namespace AWS.Deploy.CLI.Commands
+{
+    public class GenerateDeploymentProjectCommand
```

Comment:
How did you test this? we should write some test cases for this class.

Reply:
Added Integration tests and verified that they run successfully on Gamma.